src/main/java/org/spongepowered/common/util/gen/ArrayMutableBlockBuffer.java

```diff
@@ -66,16 +70,10 @@ public ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i si
         this.data = new CharBackingData(blocks);
     }
 
-    public ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i size, int[] blocks) {
-        super(start, size);
-        this.palette = palette;
-        this.data = new IntBackingData(blocks);
-    }
-
     ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i size, BackingData blocks) {
         super(start, size);
         this.palette = palette;
-        this.data = blocks.copyOf();
```

Comment:
I'm talking about [this one](https://github.com/xcube16/SpongeCommon/blob/7621cc4a8028c385830ecfcd8a10350434c41e10/src/main/java/org/spongepowered/common/util/gen/ArrayImmutableBlockBuffer.java#L53), I'm not sure if its even used but any usages of it should be changed for consistency.